graphql-dgs-client/src/test/kotlin/com/netflix/graphql/dgs/client/ErrorsTest.kt

```diff
@@ -178,6 +179,7 @@ class ErrorsTest {
     }
 
     @Test
+    @Disabled("Broken by Jackson 2.17 https://github.com/FasterXML/jackson-databind/issues/4508")
```

Comment:
There's a workaround in that linked issue that we could apply instead:

https://github.com/FasterXML/jackson-databind/issues/4508#issuecomment-2107622837

Reply:
@kilink From what I understand from the issue is that the workaround is for the Jackson folks to do, it's not something we can do as a user of Jackson.